diff --git a/node_modules/pinia/dist/pinia.esm-bundler.js b/node_modules/pinia/dist/pinia.esm-bundler.js
index 4befc54..8edd685 100644
--- a/node_modules/pinia/dist/pinia.esm-bundler.js
+++ b/node_modules/pinia/dist/pinia.esm-bundler.js
@@ -136,7 +136,7 @@ const isMacOSWebView = /*#__PURE__*/ (() => /Macintosh/.test(_navigator.userAgen
 const saveAs = !IS_CLIENT
     ? () => { } // noop
     : // Use download attribute first if possible (#193 Lumia mobile) unless this is a macOS WebView
-        'download' in HTMLAnchorElement.prototype && !isMacOSWebView
+        typeof HTMLAnchorElement !== 'undefined' && 'download' in HTMLAnchorElement.prototype && !isMacOSWebView
             ? downloadSaveAs
             : // Use msSaveOrOpenBlob as a second approach
                 'msSaveOrOpenBlob' in _navigator
diff --git a/node_modules/pinia/index.mjs b/node_modules/pinia/index.mjs
index 4befc54..e3f8e6a 100644
--- a/node_modules/pinia/index.mjs
+++ b/node_modules/pinia/index.mjs
@@ -136,7 +136,7 @@ const isMacOSWebView = /*#__PURE__*/ (() => /Macintosh/.test(_navigator.userAgen
 const saveAs = !IS_CLIENT
     ? () => { } // noop
     : // Use download attribute first if possible (#193 Lumia mobile) unless this is a macOS WebView
-        'download' in HTMLAnchorElement.prototype && !isMacOSWebView
+        typeof HTMLAnchorElement !== 'undefined' && 'download' in HTMLAnchorElement.prototype && !isMacOSWebView
             ? downloadSaveAs
             : // Use msSaveOrOpenBlob as a second approach
                 'msSaveOrOpenBlob' in _navigator
