diff --git a/node_modules/pinia/dist/pinia.mjs b/node_modules/pinia/dist/pinia.mjs
index 238dcdf..5d76561 100644
--- a/node_modules/pinia/dist/pinia.mjs
+++ b/node_modules/pinia/dist/pinia.mjs
@@ -136,7 +136,7 @@ const isMacOSWebView = /*#__PURE__*/ (() => /Macintosh/.test(_navigator.userAgen
 const saveAs = !IS_CLIENT
     ? () => { } // noop
     : // Use download attribute first if possible (#193 Lumia mobile) unless this is a macOS WebView
-        'download' in HTMLAnchorElement.prototype && !isMacOSWebView
+        typeof HTMLAnchorElement === 'undefined' && 'download' in HTMLAnchorElement.prototype && !isMacOSWebView
             ? downloadSaveAs
             : // Use msSaveOrOpenBlob as a second approach
                 'msSaveOrOpenBlob' in _navigator
